engine/crates/engine-v2/auth/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "engine-v2-auth"
+version = "0.1.0"
```

Comment:
Not important, but I think `version` is optional as of rust 1.75 so we could leave this out.